javascript-modules-library/tsconfig.json

```diff
@@ -26,8 +26,10 @@
          },
     },
     "include": [
-       "src/**/*.js",
-       "src/**/*.mjs"
+        "src/**/*.js",
+        "src/**/*.mjs",
+        "src/**/*.ts",
+        "target/types/globals.d.ts"
```

Comment:
Do we actually need this one?

Reply:
This is the only type declaration that is ambient (all other types in target/types are modules), it needs to be explicitely referenced when outside the working directory

Reply:
just for my own understanding: what happens then if not listed here? 🤔

Reply:
Done in [f21663e](https://github.com/Jahia/javascript-modules/pull/52/commits/f21663e3e7a0e8a3632421625c19a92bff1e588d)